md2conf/converter.py

```diff
@@ -250,49 +306,57 @@ class ConfluenceDocument:
     options: ConfluenceDocumentOptions
     root: ET.Element
 
-    def __init__(self, path: str, options: ConfluenceDocumentOptions) -> None:
+    def __init__(self, path: str, options: ConfluenceDocumentOptions, page_metadata: dict[str, str] = dict()) -> None:
         self.options = options
-
         path = os.path.abspath(path)
 
         with open(path, "r") as f:
             html = markdown_to_html(f.read())
 
         # extract Confluence page ID
-        page_id, html = _extract_value(
-            r"<!--\s+confluence-page-id:\s*(\d+)\s+-->", html
-        )
+        page_id, html = extract_value(r"<!--\s+confluence-page-id:\s*(\d+)\s+-->", html)
         if page_id is None:
             raise DocumentError(
                 "Markdown document has no Confluence page ID associated with it"
             )
         self.page_id = page_id
 
         # extract Confluence space key
-        self.space_key, html = _extract_value(
+        self.space_key, html = extract_value(
             r"<!--\s+confluence-space-key:\s*(\w+)\s+-->", html
         )
 
         # parse Markdown document
         if self.options.generated_by:
             content = [
                 '<ac:structured-macro ac:name="info" ac:schema-version="1">',
-                "<ac:rich-text-body><p>This page has been generated with a tool.</p></ac:rich-text-body>",
+                '<ac:rich-text-body><p> Do Not Edit In Confluence: This page is being periodically imported from the <a href="https://infra-wiki.onemedical.io/">infra-wiki</a> with a tool.</p></ac:rich-text-body>',
```

Comment:
I think you should revert to the original text here. However, we can think of options how to configure what appears here.

Reply:
I have already thought of that and am already preparing if you don't mind.

Reply:
To the contrary, I would most appreciate it. As for the Python code, the text could be passed as another configuration option. As for the command-line interface to `md2conf`, the text should likely come from a configuration file, preferably JSON, because it's a longer piece, and putting that all as a command-line parameter is likely not that practical. We don't have an option to pass a configuration file to `md2conf` at the moment, and then parse it, but that could be added later. Both of these, in my opinion, are out of scope for this pull request, and should be done in a separate one.

However, we need to revert adding a reference to an internal wiki site here such that I can merge this pull request.

Reply:
... let's discuss this here https://github.com/hunyadi/md2conf/pull/7